### PR TITLE
[Form] Fix author tag + exception messages

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

spotted in #31195 :sweat_smile:

### CODE REVIEW

**Comment by ro0NL** [comment]
while at it, i applied the same review from @javiereguiluz (https://github.com/symfony/symfony/pull/31195#pullrequestreview-230719997) here :angel:

File: src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeZoneToStringTransformer.php [hunk 1]
```diff
@@ -39,14 +39,14 @@ public function transform($dateTimeZone)
 
         if ($this->multiple) {
             if (!\is_array($dateTimeZone)) {
-                throw new TransformationFailedException('Expected an array.');
+                throw new TransformationFailedException('Expected an array of \DateTimeZone objects.');
```
Comment (kunicmarko20) [inline src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeZoneToStringTransformer.php]: the title of the pr should probably change then.

Comment (ro0NL) [reply src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeZoneToStringTransformer.php]: sure :D

**Review by OskarStark — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @ro0NL.

**Pull request merged**